integrated_channels/canvas/apps.py

```diff
@@ -12,3 +12,5 @@ class CanvasConfig(AppConfig):
     """
     name = 'integrated_channels.canvas'
     verbose_name = "Enterprise Canvas Integration"
+    oauth_token_auth_path = "login/oauth2/token"
+    oauth_redirect_uri = "urn:ietf:wg:oauth:2.0:oob"  # TODO- the uri needs to link to Django admin (?)
```

Comment:
so is the plan here to use this out of band uri for now and change it to our prod value (using env vars I presume for different envrions), in a future round of work after the endpoint is ready?

Reply:
Even if this is temporary maybe let's add a doc link somewhere about what this urn is and why we are using it here?

Reply:
This is the out of band URI because I found that when using it, you didn't have to match the user's indicated redirect url, which was useful for testing. Until the `oauth/complete` endpoint work is done, I've found it useful just to use this uri. I'll add a doc snippet explaining why it's here!

Reply:
🤦 updates, redirect uri isn't required any more because we switched to refresh tokens, so I'm removing it from the config.